src/lib/journey/stages/registration.svelte

```diff
@@ -71,7 +72,7 @@
         {firstInvalidInput}
       />
     {/each}
-    <Button width="full" style="primary" type="submit">
+    <Button busy={loading} style="primary" type="submit" width="full">
```

Comment:
We might want to deactivate the button while it's loading, but we'd probably also want to add a timer or something ... not sure. I really don't like deactivating buttons if I don't have to.